documentation/src/test/java/org/hibernate/search/documentation/mapper/pojo/standalone/entrypoints/StandalonePojoReadPropertiesFromFileIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.documentation.mapper.pojo.standalone.entrypoints;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.hibernate.search.documentation.testsupport.BackendConfigurations;
+import org.hibernate.search.documentation.testsupport.TestConfiguration;
+import org.hibernate.search.engine.cfg.spi.AllAwareConfigurationPropertySource;
+import org.hibernate.search.mapper.pojo.standalone.entity.SearchIndexedEntity;
+import org.hibernate.search.mapper.pojo.standalone.mapping.CloseableSearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.session.SearchSession;
+import org.hibernate.search.util.impl.integrationtest.common.TestConfigurationProvider;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class StandalonePojoReadPropertiesFromFileIT {
+
+	private CloseableSearchMapping theSearchMapping;
+
+	@Rule
+	public TestConfigurationProvider configurationProvider = new TestConfigurationProvider();
+
+	@Before
+	public void setup() throws IOException {
+		try ( // tag::setup[]
```

Comment:
I'd include the `try` in the example, so that indentation won't seem wrong.

Ideally you'd also move the declaration of `CloseableSearchMapping searchMapping` out of the `try` block, to show that the input stream doesn't need to remain open forever.